clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -68,25 +67,27 @@ public class AzureComputeClient extends AzureBaseClient {
 
     try {
       // Usage of local variables to ease with debugging the code; keeping the content retrieved from Azure JSDK call to help with stepping through the code and inspect the values
-      List<String> publishers = client.getVirtualMachineImagesOperations()?.listPublishers(location)?.body?.collect { it.name }
+      def ops = client.getVirtualMachineImagesOperations()
```

Comment:
About a month ago when I tried to reuse the ops object I was getting random timeouts/ops object not longer being valid. Is this fixed now? Can you please confirm that you get the expected resources in the redis cache (start with an empty cache).